resources/views/bread/partials/order.blade.php

```diff
@@ -0,0 +1,23 @@
+<ol class="dd-list">
+    @foreach ($items as $result)
+        @if($nestable && $result->parent_id != $parentId)
+            @continue
+        @endif
+    <li class="dd-item" data-id="{{ $result->getKey() }}">
+        <div class="dd-handle" style="height:inherit">
+            @if (isset($dataRow->details->view))
+                @include($dataRow->details->view, ['row' => $dataRow, 'dataType' => $dataType, 'dataTypeContent' => $result, 'content' => $result->{$display_column}, 'action' => 'order'])
+            @elseif($dataRow->type == 'image')
+                <span>
+                    <img src="@if( !filter_var($result->{$display_column}, FILTER_VALIDATE_URL)){{ Voyager::image( $result->{$display_column} ) }}@else{{ $result->{$display_column} }}@endif" style="height:100px">
+                </span>
+            @else
+                <span>{{ $result->{$display_column} }}</span>
+            @endif
+        </div>
+        @if(isset($result->nestableChildren) && !$result->nestableChildren->isEmpty())
```

Comment:
So.. nestableChildren holds the child entries... do you need $testable at all? You could just check if this list is empty, right?

Reply:
I can only check if element is under the right parent if it's nestable, otherwise there is no parent id. 

I need to change parent_id that's the hard coded value I was using at start.